### PR TITLE
vy added fun fact

### PR DESCRIPTION
**Pull request by VyLanTran**
Vy Tran added school and fun fact

### CODE REVIEW

**Review by Jimmykhangnguyen — APPROVED** [review]

**Review by khoa165 — CHANGES_REQUESTED** [review]
Let's use this format:
Name
- School
- Fun fact

File: group_5.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
-Phuc Khang:
-Quy Nguyen:
-Vy Tran:
+Phuc Khang: [School], [Fun fact]
+Quy Nguyen: [School], [Fun fact]
+Vy Tran: 
+    - School: Bucknell University
+    - Fun fact: I'm a fast typer
 Linh Nguyen: Tufts , I like making emoji face like this :>> +-+ ://// hehe
```
Comment (khoa165) [inline group_5.txt]: Thanks for formatting your part. Could you also do the same for Linh so everything is consistent

**Review by khoa165 — COMMENTED** [review]
LGTM, I left a comment

**Review by khoa165 — APPROVED** [review]
LGTM

**Pull request merged**